test/src/test/kotlin/graphql/nadel/tests/next/fixtures/defer/MultipleDeferWithDifferentServiceCallsSnapshot.kt

```diff
@@ -0,0 +1,167 @@
+// @formatter:off
+package graphql.nadel.tests.next.fixtures.defer
+
+import graphql.nadel.tests.next.ExpectedNadelResult
+import graphql.nadel.tests.next.ExpectedServiceCall
+import graphql.nadel.tests.next.TestSnapshot
+import graphql.nadel.tests.next.listOfJsonStrings
+import kotlin.Suppress
+import kotlin.collections.List
+import kotlin.collections.listOf
+
+/**
+ * This class is generated. Do NOT modify.
+ *
+ * Refer to [graphql.nadel.tests.next.UpdateTestSnapshots
+ */
+@Suppress("unused")
+public class MultipleDeferWithDifferentServiceCallsSnapshot : TestSnapshot() {
+    override val calls: List<ExpectedServiceCall> = listOf(
+            ExpectedServiceCall(
+                service = "product",
+                query = """
+                | {
+                |   product {
+                |     productName
+                |     ... @defer {
+                |       productImage
+                |     }
+                |   }
+                | }
+                """.trimMargin(),
+                variables = "{}",
+                result = """
+                | {
+                |   "data": {
+                |     "product": {
+                |       "productName": "Awesome Product"
+                |     }
+                |   },
+                |   "hasNext": true
+                | }
+                """.trimMargin(),
+                delayedResults = listOfJsonStrings(
+                    """
+                    | {
+                    |   "hasNext": false,
+                    |   "incremental": [
+                    |     {
+                    |       "path": [
+                    |         "product"
+                    |       ],
+                    |       "data": {
+                    |         "productImage": null
```

Comment:
`productImage` is returning `null` here, because of the wrong data fetcher name being used.